spec/unit/puppet/type/iis_site_spec.rb

```diff
@@ -38,26 +38,6 @@
     end
   end
 
-  # TODO: fix tests below
-  context "parameter :physicalpath" do
```

Comment:
Can you explain why you removed the validation checks and tests for nil physical paths for websites? IIRC you need a physical path for a site